### DISCUSSION TITLE
Capacitor Webview with https not in http

### DISCUSSION THREAD

**Discussion by Bijit-Mondal** [ionic-team/capacitor#6621] [General]
I want to integrate https://otpless.com, but it's SDK works only with https, and that's why I am unable to work with it as capacitor works on http. How to resolve that

**Comment by jcesarmobile** [comment]
new Capacitor 5 apps use https by default, if your app was a Capacitor 4 app or older it will use http to avoid web data loss (since localStorage and other forms of web storage are bound to the url).

But if you don't mind about that web data, you can just set the scheme to https. Check `androidScheme` config option from server config object on https://capacitorjs.com/docs/config
